file-formats/sajc/type/zif_aff_sajc_v1.intf.abap

```diff
@@ -69,5 +167,14 @@ INTERFACE zif_aff_sajc_v1
       "! <p class="shorttext">Exit Classes</p>
       "! Classes which contain the exits of the job scheduling and processing
       exit_classes        TYPE ty_exit_classes,
+      "! <p class="shorttext">Parameters</p>
+      "! List of parameters
+      parameters          TYPE ty_parameter_table,
+      "! <p class="shorttext">Groups</p>
+      "! List of groups
+      groups              TYPE ty_group_table,
+      "! <p class="shorttext">Sections</p>
+      "! List of sections
+      sections            TYPE ty_section_table,
```

Comment:
We suggest changing the order:
1. Sections
2. Groups
3. Parameters

Reply:
I changed it according to the suggestion